### PR TITLE
MB-1465: Named user

### PR DESCRIPTION
**Pull request by rlepinski**

### CODE REVIEW

File: Assets/Plugins/iOS/UAUnityPlugin.m [hunk 1]
```diff
@@ -221,6 +221,17 @@ void UAUnityPlugin_disableBackgroundLocation() {
     [UAirship shared].locationService.backgroundLocationServiceEnabled = NO;
 }
 
+void UAUnityPlugin_setNamedUserID(const char *namedUserID) {
+    NSString *namedUserIDString = [NSString stringWithUTF8String:namedUserID];
+    NSLog(@"UnityPlugin setNamedUserID");
```
Comment (hcrowell) [inline Assets/Plugins/iOS/UAUnityPlugin.m]: could we also log the namedUserID pls?
ie. NSLog(@"UnityPlugin setNamedUserID %@", namedUserIDString);

**Comment by hcrowell** [comment]
:tangerine: after reviewing minor comment above ^

File: Assets/Plugins/iOS/UAUnityPlugin.m [hunk 2]
```diff
@@ -263,7 +274,11 @@ - (void) notifyRecievedPush:(NSDictionary *)notification {
 
 
 #pragma mark -
-#pragma mark Helper C Functions
+#pragma mark Helpers
+
++ (NSString *)stringOrNil:(NSString *)string {
```
Comment (hcrowell) [inline Assets/Plugins/iOS/UAUnityPlugin.m]: is this used anywhere in this PR? (looks like its used in the custom events PR https://github.com/urbanairship/ua-unity-plugin/pull/24)

**Comment by hcrowell** [comment]
:tangerine: after reviewing minor comments above ^

File: Assets/Plugins/iOS/UAUnityPlugin.m (hunk 2 above)
Comment (rlepinski) [inline Assets/Plugins/iOS/UAUnityPlugin.m]: my bad, ill remove it

**Pull request merged**